rules/DeadCode/Rector/ClassMethod/RemoveUnusedPrivateMethodRector.php

```diff
@@ -228,4 +235,31 @@ private function shouldSkipArrayCallable(Class_ $class, ?ArrayCallable $arrayCal
         // is current class method?
         return ! $this->isName($class, $arrayCallable->getClass());
     }
+
+    private function isPrivateAbstractMethodInTrait(ClassMethod $classMethod, ?string $classMethodName): bool
+    {
+        if (null === $classMethodName) {
+            return false;
+        }
+
+        $scope = $classMethod->getAttribute(AttributeKey::SCOPE);
+        if (! $scope instanceof Scope) {
+            return false;
+        }
+
+        $classReflection = $scope->getClassReflection();
+        if (null === $classReflection) {
+            return false;
+        }
```

Comment:
ClassReflection check no longer needed as already checked in shouldskip(), you should can use `@var` for it